keel-sql/src/main/kotlin/com/netflix/spinnaker/keel/sql/SqlArtifactRepository.kt

```diff
@@ -70,6 +70,15 @@ class SqlArtifactRepository(
   private val sqlRetry: SqlRetry,
   private val artifactSuppliers: List<ArtifactSupplier<*, *>> = emptyList()
 ) : ArtifactRepository {
+
```

Comment:
You're going to add columns for these in another PR, right?

Reply:
Yup! Generated columns so there's no risk of things getting out-of-sync.

Reply:
ah yes, i see your earlier comment now